opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -95,6 +92,28 @@ public synchronized void reset() {
         return new ArrayList<>(this.finishedSpans);
     }
 
+    /**
+     * @return a copy of all finish()ed Traces(Spans) started by this MockTracer grouped by traceId and spanId in HashMap format.
+     */
+    public Map<String, Map<String, Span>> finishedTraces() {
+        Map<String, Map<String, Span>> result = new HashMap<>();
```

Comment:
Suggestion to use `LinkedHashMap` to keep insertion order from the original `List` ?

(both `new HashMap` calls)

Reply:
Brilliant!